docs/guides/options.md

```diff
@@ -1,138 +1,351 @@
 
-### loop ###
-The loop attribute causes the video to start over as soon as it ends. This could be used for a visual effect like clouds in the background.
-```html
-<video loop ...>
-or
-{ "loop": true }
+```js
+// The following code creates a player with ONLY bigPlayButton and
+// controlBar child components.
+videojs('my-player', {
+  children: [
+    'bigPlayButton',
+    'controlBar'
+  ]
+});
 ```
 
+The `children` options can also be passed as an `Object`. In this case, it is used to provide `options` for any/all children, including disabling them with `false`:
 
-### width ###
-The width attribute sets the display width of the video.
-```html
-<video width="640" ...>
-or
-{ "width": 640 }
+```js
+// This player's ONLY child will be the controlBar. Clearly, this is not the
+// ideal method for disabling a grandchild!
+videojs('my-player', {
+  children: {
+    controlBar: {
+      fullscreenControl: false
+    }
+  }
+});
 ```
 
+### `${componentName}`
 
-### height ###
-The height attribute sets the display height of the video.
-```html
-<video height="480" ...>
-or
-{ "height": 480 }
+> Type: `Object`
+
+Components can be given custom options via the _lower-camel-case variant of the component name_ (e.g. `controlBar` for `ControlBar`). These can be nested in a representation of grandchild relationships. For example, to disable the fullscreen control:
+
+```js
+videojs('my-player', {
+  controlBar: {
+    fullscreenControl: false
+  }
+});
 ```
 
-Component Options
------------------
+## Tech Options
+
+### `${techName}`
+
+> Type: `Object`
+
+Video.js playback technologies (i.e. "techs") can be given custom options as part of the options passed to the `videojs` function. They should be passed under the _lower-case variant of the tech name_ (e.g. `"flash"` or `"html5"`).
 
-You can set the options for any single player component. For instance, if you wanted to remove the `muteToggle` button, which
-is a child of `controlBar`, you can just set that component to false:
+This is not used in most implementations, but one case where it may be is dictating where the Video.js SWF file is located for the `Flash` tech:
 
 ```js
-var player = videojs('video-id', {
-  controlBar: {
-    muteToggle: false
+videojs('my-player', {
+  flash: {
+    swf: '//path/to/videojs.swf'
   }
 });
 ```
 
-This also works using the `data-setup` attribute on the video element, just remember the options need to use proper JSON
-notation.
+However, this is a case where changing the global defaults is more useful:
 
-```html
-<video ... data-setup='{ "controlBar": { "muteToggle": false } }'></video>
+```js
+videojs.options.flash.swf = '//path/to/videojs.swf'
 ```
 
-The [components guide](./components.md) has an excellent breakdown of the structure of a player, you
-just need to remember to nest child components in a `children` array for each level.
+#### `nativeCaptions`
```

Comment:
This is an alias to `nativeTextTracks`. I think it should be mentioned below instead of having its own section.